### DISCUSSION TITLE
Shop

### DISCUSSION THREAD

**Discussion by Lumycea** [clicepfl/website#10] [Ideas]
Roman suggested that we could have a shop for registration to the events or buy the hoodies.
I think we should study whether it is feasible and not too costly (both in dev and admin for the invoices)

**Comment by mochiruntime** [comment]
Sounds great! We should discuss the scope of the features we'd like to implement e.g. users, logistics, maybe also payments?
Depending on the complexity, we could simply implement something on top of Strapi, or we may want to integrate third-party tools such as Stripe (https://stripe.com/en-ch) or even have a dedicated back-end service such as Medusa (https://medusajs.com/).
